utils.py

```diff
@@ -165,7 +165,10 @@ def __init__(self, url, auth, verify):
             self.curl.setopt(pycurl.HTTPAUTH, pycurl.HTTPAUTH_BASIC)
             self.curl.setopt(pycurl.USERPWD, '%s:%s' % auth)
         if verify:
-            self.curl.setopt(pycurl.CA_INFO, verify)
+            self.curl.setopt(pycurl.CAINFO, verify)
+        else:
```

Comment:
I believe these two options are all we need.

https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYHOST.html
https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYPEER.html